src/google/protobuf/descriptor.proto

```diff
@@ -80,7 +80,7 @@ enum Edition {
   EDITION_2024 = 1001;
 
   // Placeholder editions for testing feature resolution.  These should not be
-  // used or relyed on outside of tests.
```

Comment:
Unfortunately we can't accept any external contributions to this file

Reply:
Ah, this particular typo is what started this whole process for me! It leaks into downstream code that references `google/api/annotations.proto`, typically to annotate services with HTTP routing metadata.
If someone internal could "independently" fix this typo then that'd be great! 🙏